sdk/src/main/java/com/karhoo/sdk/api/service/payments/PaymentProviderInteractor.kt

```diff
@@ -36,15 +36,17 @@ internal class PaymentProviderInteractor @Inject constructor(
             is Resource.Success -> {
                 val paymentProvider = result.data
                 userManager.paymentProvider = paymentProvider
-                fetchUserCardDetails(user = userManager.user)
+                if (paymentProvider.provider.id.equals("Braintree", true) && userManager.user.organisations.isNotEmpty()) {
```

Comment:
what if it ever changes to `braintree` instead of `Braintree` because someone on the backend is careless while modifying the web-service? we should make the id lowercase here and compare it with "braintree" just in case